docs/topics/jvm/jvm-records.md

```diff
@@ -0,0 +1,74 @@
+[//]: # (title: Records)
+
+> JVM records feature is [Experimental](components-stability.md). It may be dropped or changed at any time. Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42430).
```

Comment:
In our guidelines we have a special warning for experimental features with opt-in, see [here](https://docs.google.com/document/d/1zmjZvQHsvkohMZCQTpv64gPDZEpUOQwYqQ-YH601oWc/edit#heading=h.ty9tig7yrvtm) or an example on the [Sealed interfaces](https://kotlinlang.org/docs/sealed-classes.html#sealed-interfaces) doc.

Reply:
How fast do they change! %)
I took the one from Sealed interfaces